airbyte-integrations/connectors/source-instagram/source_instagram/client/api.py

```diff
@@ -38,11 +38,13 @@
 backoff_policy = retry_pattern(backoff.expo, FacebookRequestError, max_tries=5, factor=5)
 
 
-# This function removes the _nc_rid parameter from the video url and ccb from profile_picture_url for users.
-# _nc_rid is generated every time a new one and ccb can change its value, and tests fail when checking for identity.
-# This does not spoil the link, it remains correct and by clicking on it you can view the video or see picture.
 def clear_video_url(record_data: dict = None):
-    if record_data.get("media_type") == "VIDEO":
+    """
+    This function removes the _nc_rid parameter from the video url and ccb from profile_picture_url for users.
+    _nc_rid is generated every time a new one and ccb can change its value, and tests fail when checking for identity.
+    This does not spoil the link, it remains correct and by clicking on it you can view the video or see picture.
+    """
+    if record_data.get("media_type") == "VIDEO" and record_data.get("media_url"):
```

Comment:
Is the goal to remove all query string Params or just this one? 

If there are other query params after this one in the URL they will also be removed. Is this intentional? 

In either case, wouldn't it be easier to use urllib.parse or something similar to properly remove url params instead of doing string editing?